proto/interchain_security/ccv/v1/wire.proto

```diff
@@ -88,6 +77,12 @@ enum ConsumerPacketDataType {
       [ (gogoproto.enumvalue_customname) = "VscMaturedPacket" ];
 }
 
+// Note this type is used during IBC handshake methods for both the consumer and provider
```

Comment:
nit: slight inconsistency in the usage of `Note` in the same file, e.g., below it's ` // NOTE: ccv.InfractionType to ...`.

Reply:
https://github.com/cosmos/interchain-security/pull/1071/commits/1674152eb664d5e0319ce6ba3ea820c54edae088